src/reducer/slices/collections.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../async/queries';
 import { Nft, AssetContract } from '../../lib/nfts/queries';
 import config from '../../config.json';
-
+const cobj = (config as {[key: string]: any})[`${process.env.REACT_APP_NETWORK}`];
```

Comment:
We should more rigorously represent the shape of the config at the type level, and avoid `any` when at all possible.

If my intuition in the above comment is correct, and you're proposing a change in which the environment configs are placed under a single file and namespaced by environment, something like the following would suffice:
```typescript
type Env = 'sandbox' | 'mainnet' | 'testnet';

type EnvConfig = {
  rpc: string;
  network: string;
  bcd: {
    api: string;
    gui: string;
  }
  // ...
};

type EnvConfigMap = Record<Env, EnvConfig>;
```